Added a api call to lambda/update for upgrading the contract. (#68)
PassbirdCo/MetaKeepCodeSamples: lambda/custom-erc721-upgradeable/README.md

npm run upgradeToCustomERC721UpgradeableV2
```

> _NOTE:_ After the new Lambda is deployed and Upgraded, you also need to update the ABI by calling lambda/update API.
> 

Reviewer: ```suggestion
This will upgrade the contract code and also update the ABI by making an API call to `/v2/lambda/update`.

```
